### DISCUSSION TITLE
Restoring a previous checkpoint

### DISCUSSION THREAD

**Discussion by Oncorporation** [d8ahazard/sd_dreambooth_extension#738] [Q&A]
In the dreambooth model folder we now have checkpoints, but how do we start training again from a previous checkpoint or create a ckpt from a previous checkpoint.

These files are found
/models/dreambooth/[modelname]/checkpoints

and are listed as
checkpoint-436
checkpoint-872
checkpoint-1308

**Comment by kernalkue** [comment]
@Oncorporation - did you find an answer to this?
